samples/OnFido-Combined/Policies/TrustFrameworkBase.xml

```diff
@@ -450,8 +450,8 @@
             <Item Key="UserMessageIfOldPasswordUsed">Looks like you used an old password</Item>
 
             <Item Key="ProviderName">https://sts.windows.net/</Item>
-            <Item Key="METADATA">https://login.microsoftonline.com/{tenant}/.well-known/openid-configuration</Item>
-            <Item Key="authorization_endpoint">https://login.microsoftonline.com/{tenant}/oauth2/token</Item>
+            <Item Key="METADATA">https://login.microsoftonline.com/{your_tenantId}/.well-known/openid-configuration</Item>
```

Comment:
These are special identifiers {tenant}, you shouldn't actually need to populate these at all.